sensors/06-list-datasets-by-plot.Rmd

```diff
@@ -7,25 +7,26 @@ from the workbench or Globus.
 ## Getting started
 
 After installing terrautils, you should be able to import the *product* module.
-```
+```{python}
```

Comment:
> After installing terrautils

how do you install terrautils?

Reply:
terrautils could also be pre-installed ... I drafted this for the [Rstudio dockerfile](https://github.com/terraref/toolserver-dockerfiles/blob/master/rstudio/Dockerfile) based on the terrautils docker file but haven't tested it:

```
# Install terrautils
RUN cd && git clone https://github.com/terraref/terrautils && cd terrautils 
COPY setup.py requirements.txt MANIFEST.in /tmp/terrautils/
RUN pip install --upgrade  -r /tmp/terrautils/requirements.txt

COPY terrautils /tmp/terrautils/terrautils
RUN pip install --upgrade /tmp/terrautils \
&& rm -rf /tmp/terrautils
```